package.json

```diff
@@ -76,6 +76,7 @@
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-markdown": "^2.2.1",
     "eslint-plugin-node": "^11.1.0",
```

Comment:
Shouldn't this be a dependency of `@remix-run/eslint-config` as we're now having extending `@remix-run/eslint-config/node` in the configs?

Reply:
It already is. https://github.com/remix-run/remix/blob/main/packages/remix-eslint-config/package.json#L28